src/appkit.cs

```diff
@@ -25489,11 +25498,43 @@ interface NSAccessibilityTable : NSAccessibilityGroup {
 	}
 
 	[Mac (10,10)]
+	[Protocol]
 	interface NSAccessibilityOutline : NSAccessibilityTable {
```

Comment:
normally that would break things... but since there's no `[BaseType]` I have the feeling that this was never generated

Reply:
It was never, correct.